frontend/.env.production

```diff
@@ -0,0 +1 @@
+REACT_APP_CODALAB_RECAPTCHA_SITE_KEY=6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI
```

Comment:
We should .gitignore this file

Reply:
Yes I though it was in there already but it only has .`env`.